addons/60fps/addon.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "60FPS",
+  "description": "Ctrl+Click the green flag to toggle 60fps.",
+  "credits": [
+    {
+      "name": "Jeffalo"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "tags": [],
```

Comment:
Tags: editor? It could also be considered community, tbh, but it's closer to editor.